internal/pkg/runner/job/job_runner.go

```diff
@@ -0,0 +1,88 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+//Package job provides support for invoking scheduled jobs
+
+package job
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/copilot-cli/internal/pkg/aws/cloudformation"
+	"github.com/aws/copilot-cli/internal/pkg/aws/stepfunctions"
+	"github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation/stack"
+)
+
+type jobExecutor interface {
+	Execute(stateMachineARN string) error
+}
+
+type StackRetriever interface {
+	StackResources(name string) ([]*cloudformation.StackResource, error)
+}
+
+type JobRunner struct {
```

Comment:
Can we add doc comments to all exported names please 🥺 
https://github.com/golang/go/wiki/CodeReviewComments#doc-comments

Reply:
I added comments, but not too sure of the wording, feel free to edit them